locales/en.json

```diff
@@ -1990,6 +1990,8 @@
 				"printSubmission": "Print submission",
 				"printTask": "Print task definition",
 				"submitted": "Submitted",
+				"statsSubmitted": "'{{submissionCount}}' out of {{totalCount}} submitted",
```

Comment:
why in english version {{submissionCount}} and {{gradedCount}} are in quotes, but in german are not?

Reply:
fixed